extractors/python_functions/credit_card_extraction/code_snippet.md

```diff
@@ -1,11 +1,10 @@
 ```python
 import re
-from typing import List, Tuple
 
 YOUR_ATTRIBUTE: str = "text" # only text attributes
 YOUR_LABEL: str = "cardNumber"
 
-def credit_extractor(record) -> List[Tuple[str, int, int]]:
+def credit_extractor(record):
```

Comment:
creditcard_extraction sounds more professional

Reply:
It also matches the naming of the rest of the module. Changed it!